app/views/posts/index.html.erb

```diff
@@ -1 +1,38 @@
-<h1>See my posts</h1>
\ No newline at end of file
+<div class="main">
+  <div class="content">
+    <h1><%= @user.name %>'s Posts</h1>
+    <div class='box flex'>
+      <div class='img flex'>
+        <img src="<%= @user.photo %>" alt="unsplash">
+      </div>
+      <div class='user-details'>
+        <p><%= @user.name %></p>
+          <span class="post-no">number of posts: <%= @user.posts_counter %></span>
+      </div>
+    </div>
+
+    <% @posts.each do |post| %>
+      <div class="post">
+        <p><b><%= post.title %></b></p>
+        <p><%= post.text %></p>
+        <div class="come_like">
+          <span>comments: <%= post.comments_counter %>, likes: <%= post.likes_counter %></span>
+        </div>
+        <div class="comments">
```

Comment:
- [ **OPTIONAL** ] **IMPORTANT:** I think it would be better you add a feature when clicking on each `Post` takes you to its show page as shown in the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_page.png) as you did for a `User`, You can apply this approach in every page you have a `Post`